cli/flox/src/commands/envs.rs

```diff
@@ -202,7 +202,7 @@ fn get_registered_environments(
         let pointer = entry.latest_env()?.pointer.clone();
 
         // If we have a path registered that has since been deleted, skip it
-        if !entry.path.exists() {
+        if !entry.exists() {
```

Comment:
suggestion nonblocking: I think it would be better to `prune_nonexistent` and then drop this conditional. I think the only latency that would add would be acquiring the lock and writing the file, which seems like low enough latency it shouldn't make much difference